app/models/answer.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Answer < ApplicationRecord
+    validates :title, presence: true
+  
+    belongs_to :question
+    has_many :player_answers, dependent: :destroy
+end
```

Comment:
You should always make sure your files end with a new line. This is because [POSIX defines a line as](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)
> A sequence of zero or more non- \<newline\> characters plus a terminating \<newline\> character.
meaning that some applications will fail to read the last line of the file if it is not terminated by a newline character.

Reply:
I think I have fixed the inconsistency with the new lines